### ISSUE TITLE
Different font

### ISSUE THREAD

**Issue by guytenaerefe** [ThirdHandAlgorithms/PlanetaryDeception#41]
Or do you really like this Font? Up for discussion. If we change the font, start lines with capitals.

**Comment by partouf** [comment]
I agree, but it's not easy to find them. I got the current one from a free font website. @ http://www.dafont.com/blade-runner.font

**Comment by guytenaerefe** [comment]
Thanks for the link, I suspected as much. We'll see!
